tests/llmobs/test_llmobs_span_writer.py

```diff
@@ -1,93 +1,76 @@
-import os
 import time
 
 import mock
 import pytest
 
+from ddtrace import config
 from ddtrace.llmobs._writer import LLMObsSpanWriter
 from tests.llmobs._utils import _chat_completion_event
 from tests.llmobs._utils import _completion_event
 
 
-INTAKE_ENDPOINT = "https://llmobs-intake.datad0g.com/api/v2/llmobs"
-DD_SITE = "datad0g.com"
-dd_api_key = os.getenv("DD_API_KEY", default="<not-a-real-api-key>")
+INTAKE_URL = "https://llmobs-intake.%s" % config._dd_site
```

Comment:
Can we continue to use staging `datad0g.com` for testing?
```suggestion
DD_SITE = "datad0g.com"
INTAKE_ENDPOINT = "https://llmobs-intake.{}/api/v2/llmobs".format(DD_SITE)
```

Reply:
this change is because we no longer pass in datadog_site as a parameter to the constructor, since we use the same class for both agent and agentless, and pull datadog site from config directly.

I could support optionally passing in datadog site via a constuctor parameter? that would let us pass in `datad0g.com` for testing.

alternatively, if there's a way to set the config values for unit tests, we could do that instead

Reply:
No problem, we have the `override_global_config()` test utility to set config values for tests: 
```python
from tests.utils import override_global_config

with override_global_config(dict(_dd_site="datad0g.com")):
    # rest of test
```